README.md

```diff
@@ -161,6 +161,12 @@
   * Module: `code_challenges/code_wars/change_it_up.py`
   * Tests: `tests/code_wars/test_change_it_up.py`
   * URL: [challenge url](https://www.codewars.com/kata/58039f8efca342e4f0000023/python)
+<<<<<<< HEAD
```

Comment:
Always double check the files in github after a conflit resolution to ensure things look correct